### PR TITLE
187121440 materials bulk upload

### PR DESCRIPTION
**Pull request by warrenlet**
### [Pivotal Tracker Link][tracker]

[tracker]: https://www.pivotaltracker.com/story/show/your-story-id

## What this PR does:

This pull request fixes|implements (pick one...) ______.

### How should this PR be tested?

* Is there a deploy we can view?
* What do the specs/features test?
* Are there edge cases to watch out for?

### Checklist:

- [ ] Has this been deployed to a staging environment or reviewed by a customer?
- [ ] Tag someone for code review (either a coach / team member)
- [ ] I have renamed the branch to match PivotTracker's suggested one (necessary for BlueJay)

### CODE REVIEW

**Comment by Edge7481** [comment]
Would it be possible to write a few tests that checks the csv is working

**Review by fp456 — CHANGES_REQUESTED** [review]
Wondering if it'll be possible to add some tests for the code added. Also, please update the PR template with the given info, thank you!

File: app/views/admin/events/index.html.haml [hunk 1]
```diff
@@ -35,12 +35,13 @@
...
-              %b Event_ID:Commercial_Link
+              Upload your file.csv with data in the following format:
+              %b Event_ID,Title,URL
               for instance:
               %pre
-                11:https://youtube.com/myvideo
-                23:https://vimeo.com/myvideo
+                Event_ID,Title,URL
+                1,title,https://youtube.com/myvideo
+                2,title,https://vimeo.com/myvideo
```
Comment (cycomachead) [inline app/views/admin/events/index.html.haml:44]:
```suggestion
                1,Session Recording,https://youtube.com/myvideo
                2,Demo Video,https://vimeo.com/myvideo
```

**Review by fp456 — APPROVED** [review]
